indico/modules/events/papers/templates/emails/added_to_reviewing_team.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'events/papers/emails/base.html' %}
+
+{% block subject_message %}Added in paper reviewing team{% endblock %}
+
```

Comment:
Doesn't it look less strange without empty lines between one-line blocks? Probably just a matter of personal taste though..